test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -10,7 +10,7 @@
     collectMetrics: true
   workloads:
   - name: 500Nodes
-    labels: [fast]
+    labels: [integration-test, fast]
```

Comment:
What's the rationale for which test should be integration-test and others are not?